mapper/tedge_mapper/src/converter.rs

```diff
@@ -2,4 +2,16 @@ pub trait Converter: Send + Sync {
     type Error;
 
     fn convert(&self, input: &str) -> Result<String, Self::Error>;
+
+    fn convert_child_device_payload(
+        &self,
+        _input: &str,
+        _child_id: &str,
```

Comment:
Input parameters in a trait function prefixed with _ looks weird? Is it because the default implementation is ignoring it, which might be throwing a warning?

Reply:
Exactly. Compiler warns unused variables because the default implementation doesn't use them.